<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCM</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="2">
     <widget class="QComboBox" name="plot_type_cb">
      <item>
       <property name="text">
        <string>Pressure</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mass</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mass Sub</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half+Full Cycle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half Cycle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half+Full Cycle (Mass Only)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half Cycle (Mass Only)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half+Full Cycle Density</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Half+Full Cycle Density (Mass Only)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="ax_cb">
      <item>
       <property name="text">
        <string>Left Ax</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Right Ax</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="cols_plot_pb">
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="fill_cols_pb">
      <property name="text">
       <string>Fill Columns</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Select Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="1">
         <widget class="QTreeWidget" name="treeWidget_pressure">
          <column>
           <property name="text">
            <string>Pressure</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QTreeWidget" name="treeWidget_mass">
          <column>
           <property name="text">
            <string>Mass</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTreeWidget" name="treeWidget_time">
          <column>
           <property name="text">
            <string>Time</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTreeView" name="QCMtreeView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Params_Tab">
       <attribute name="title">
        <string>Params w/ Pressure</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="12" column="1">
         <widget class="QDoubleSpinBox" name="adp_time">
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="value">
           <double>61.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="wait">
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="value">
           <double>50.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="num_A">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="12" column="3">
         <widget class="QDoubleSpinBox" name="bdp_time">
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="value">
           <double>61.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="13" column="1">
         <widget class="QDoubleSpinBox" name="start_time">
          <property name="maximum">
           <double>99999.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>% Time</string>
          </property>
         </widget>
        </item>
        <item row="13" column="3">
         <widget class="QSpinBox" name="num_exp">
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>A D+PTime (s)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="wait_time_label">
          <property name="text">
           <string>Wait (s)</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="4">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;Use this section for anything with (mass only) Find the appoximate start time.  A D+P is the A precursor dose + purge time. this needs to be exact,  same for B. The program will find the mass at the interval for the number of exposures &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="P_Threshold">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="singleStep">
           <double>0.001000000000000</double>
          </property>
          <property name="value">
           <double>0.050000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="numa_label">
          <property name="text">
           <string># of A</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="pthress_label">
          <property name="text">
           <string>Pressure</string>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Start Time (s)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="numb_label">
          <property name="text">
           <string># of B</string>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>B D+P Time (s)</string>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Num of Exposures</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Den. (g/cm3)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>From To Time (s)</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Refresh Params</string>
          </property>
         </widget>
        </item>
        <item row="8" column="3">
         <widget class="QDoubleSpinBox" name="To_Time">
          <property name="maximum">
           <double>99999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
          <property name="value">
           <double>9999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QDoubleSpinBox" name="From_Time">
          <property name="maximum">
           <double>9999999999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QDoubleSpinBox" name="Density">
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="time_through_purge">
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.020000000000000</double>
          </property>
          <property name="value">
           <double>0.880000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QSpinBox" name="num_B">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <widget class="QTextBrowser" name="textBrowser_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>150</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt;&quot;&gt;Use this section for depositions where the pressure is readable.  # of A and # of B is the number of A and B doses in succession.  Pressure is a setpoint, if the pressure goes up by a certain amount,  it will count as a dose.  % time is saying how far through the purge do you want to count the mass as the mass of the half-cycle.  Wait is a way of eliminating noise by saying the next exposure wont occur until at least the wait time&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="4">
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QComboBox" name="time_option">
          <item>
           <property name="text">
            <string>From:To Time</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Plot Limits</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Integrate Pressure</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Num B</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDoubleSpinBox" name="waittime_sb">
          <property name="value">
           <double>28.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="threshold_sb">
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
          <property name="value">
           <double>0.020000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Threshold (torr)</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Wait Time (s)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="intmode_rb">
          <property name="text">
           <string>Integrate</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Num A</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="num_a_sb"/>
        </item>
        <item row="4" column="3">
         <widget class="QSpinBox" name="num_b_sb"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>A Dose+Purge Time</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="adptime_sb"/>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>B Dose+Purge Time</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QDoubleSpinBox" name="bdptime_sb"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="start_pb">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="end_pb">
          <property name="text">
           <string>End</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="2">
         <widget class="QLabel" name="ave_mcpc_label">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Ave Mass Change B</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="ave_mcpahc_label">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string># Doses</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="ave_mcpbhc_label">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="num_doses">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Ave Mass Change A</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QTextEdit" name="textEdit"/>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Average MCPC</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QTextBrowser" name="textBrowser_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:36pt; font-weight:600; font-style:italic; text-decoration: underline;&quot;&gt;UNDER WORK&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
